src/app/component/PaypalCheckout/PaypalTypes.ts

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
should we contain the type here, or should it eventually go into the schemes folder as well?

Reply:
Moving to scheme folder would be good. Especially once payments are integrated with firebase